NGCHM/WebContent/javascript/PdfGenerator.js

```diff
@@ -32,12 +32,22 @@ PDF.canGeneratePdf = function() {
 	return SUM.isVisible() || DVW.anyVisible();
 };
 
+PDF.pdfDialogClosed = function() {
```

Comment:
Shouldn't "PDF Dialog" be modal?  With a function like jquery's modaldialog(), this logic isn't needed. (?)